integrated_channels/canvas/client.py

```diff
@@ -265,7 +276,114 @@ def delete_course_completion(self, user_id, payload):  # pylint: disable=unused-
         # Todo: There isn't a great way for users to delete course completion data
         pass
 
+    def cleanup_duplicate_assignment_records(self, courses):
+        """
+        Placeholder
+        """
+        self._create_session()
```

Comment:
mind dropping a line or two explaining the quick algorithm idea here? like how a duplicate assignment is being detected may be sufficient

Reply:
It was more clear to me to add comments through the process to clarify steps taken. The doc string gives a high level over view of what's being done.